portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,14 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  font-family: sans-serif;
 }
-
+ 
+#p {
```

Comment:
Is this supposed to be just `p {`?
I don't see any elements with id of p in the html
To select all p elements, `p {` will work
See https://www.w3schools.com/css/css_selectors.asp if you would like to know more about selectors

Reply:
Done!